### DISCUSSION TITLE
Login Problem

### DISCUSSION THREAD

**Discussion by MikyCage** [chromiecraft/chromiecraft#2252] [General]
So, I get into the game, and it says it wants to update something, if I allow it, nothing happens, for a good 10-15 minutes then it says some patches need installed and then dosen't work. What do I do?

**Comment by Amrx2** [comment]
Hi Mikycage, you need to ensure you are logging in with your username, not your email. If the client has tried to update you might need to reinstall a fresh client in order to login.

**Reply by MikyCage** [reply]
Thanks, I'll try that!
